scripts/install/deploy/install_db.sh

```diff
@@ -28,3 +28,39 @@ fi
 
 sudo cp timeless_pg.service /lib/systemd/system/
 sudo systemctl start timeless_pg.service
+
+#checks if the database exists and create if not
+if [! psql -lqt | cut -d \| -f 1 | grep -qw "timelessdb"; ]; then
+    echo "Creating database: timelessdb"
+    sudo -u postgres psql -c "CREATE DATABASE timelessdb;"
+fi
+
+#checks if the user exists and create if not
+if [! psql -t -c '\du' | cut -d \| -f 1 | grep -qw "timeless_user"; ]; then
+    echo "Creating user: timeless_user"
```

Comment:
@paulodamaso can you add datetime to these echos? you did it already in the backup scripts, if I remember well, should be easy to take it from there